config/features.yml

```diff
@@ -72,6 +72,9 @@ features:
   hca_submission_failure_email_va_notify:
     actor_type: user
     description: Send HCA Submission Failure email using VANotify.
+  cg1010_oauth_2_enabled:
```

Comment:
New Flipper toggle so we can validate this in staging before enabling it in production. All of these changes are behind this toggle.